foundations/foundation-metrics/src/main/java/io/servicecomb/foundation/metrics/MetricsServoRegistry.java

```diff
@@ -117,9 +124,18 @@ public void initMetricsPublishing() {
     String fileName = DynamicPropertyFactory.getInstance().getStringProperty(FILENAME, "metrics").get();
     String filePath = DynamicPropertyFactory.getInstance().getStringProperty(FILEPATH, ".").get();
     MetricObserver fileObserver = new FileMetricObserver(fileName, new File(filePath));

```

Comment:
move observer out of registry, who use observer who create it?

Reply:
Fixed